exercises/grade-school/src/test/java/SchoolTest.java

```diff
@@ -12,7 +12,6 @@
 import org.hamcrest.Matcher;
 import org.hamcrest.collection.IsIterableContainingInOrder;
 import static org.hamcrest.CoreMatchers.*;
-import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertEquals;
```

Comment:
I think quite a few of these imports aren't needed any more? Feel free to remove them as part of this PR (ctrl + alt + O should remove unused imports using IntelliJ, see https://www.jetbrains.com/help/phpstorm/optimizing-imports.html), or I can open a new issue for it if you'd prefer 🙂

Reply:
I took care of it, I also removed some redundant initializations (of `school`) in every test since it's already done in `@Before`.